CHANGELOG.md

```diff
@@ -88,7 +88,6 @@ Version number updated to 3.4 to make numbering consistent with Mattermost serve
 ### Known Issues
 
 #### Windows
- - Copying a link address and pasting it inside the app doesn't work
```

Comment:
I think we should still keep it in the v3.4 changelog as it'll continue to be a bug in that version.

We should just ensure it's removed in the next version's changelog